release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1496,6 +1496,8 @@ In the following tables, features are marked with the following statuses:
 
 * Specifying a standard Ebdsv5 or Ebsv5 family machine type instance is not supported when installing an Azure cluster. This limitation is the result of the Azure terraform provider not supporting these machine types. (link:https://issues.redhat.com/browse/OCPBUGS-18690[*OCPBUGS-18690*])
 
+* When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
```

Comment:
Is it appropriate to give the KCS articles link to use as a workaround. Because oc binary provided by the cluster still may refer to RHEL8 oc?

Reply:
I mean this article https://access.redhat.com/solutions/7046917

Reply:
@ardaguclu We could, but I didn't find any other information in the KCS article that I thought was helpful. The bug listed in the KCS is a security level that non-RH people can't see. And I didn't see any other workaround proposed in the KCS other than using the binary that came with the cluster. Am I missing something?

Reply:
Yes, you are right. Let's leave this as is. Thanks.

Reply:
Okay cool. Might be worthwhile to update the KCS to use a bug that is public at some point too.